src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md

```diff
@@ -9,6 +9,9 @@ functional_areas:
 
 The Remote Storage module provides the option to store media files and schedule imports/exports in a persistent, remote storage container using a storage service, such as AWS S3 or Azure Blob Storage. By default, Magento stores media files in the same filesystem that contains the application. This is inefficient for complex, multi-server configurations, and can result in degraded performance when sharing resources. With the Remote Storage module, you can store media files in the `pub/media` directory and import/export files in the `var` directory of the remote object storage to take advantage of server-side image resizing.
 
+{:.bs-callout-info}
+The AWS S3 object storage is available since the Magento 2.4.2 version only.(See: [Magento Open Source 2.4.2 Release Notes]{{page.baseurl}}/release-notes/open-source-2-4-2.html/)
```

Comment:
```suggestion
Remote storage is available in version 2.4.2 and later only. See the [2.4.2 release notes]{{page.baseurl}}/release-notes/open-source-2-4-2.html).
```

Reply:
Thank you a lot @jeff-matthews,
I have implemented your suggestion, please review :)